Framework/PythonInterface/test/python/mantid/_testing/AssertAlmostEqualTest.py

```diff
@@ -22,7 +22,7 @@ def setUpClass(self):
         self.ws3 = ws3
 
     def test_simple(self):
-        assert_almost_equal(self.ws1, self.ws1)
+        assert_almost_equal(self.ws1, self.ws2, atol=1, rtol=1)
```

Comment:
I like that you test having both specified. Please add a test for specifying only `atol` and only `rtol`

Reply:
rtol and atol tests existed initially.